add-on/chrome/bootstrap.js

```diff
@@ -1404,10 +1404,8 @@ function startup(data) {
     return;
   }
 
-  // register about: handlers
-  AboutLoop.conversation.register(); // eslint-disable-line no-undef
-  AboutLoop.panel.register(); // eslint-disable-line no-undef
-  AboutLoop.toc.register(); // eslint-disable-line no-undef
+  Services.ppmm.loadProcessScript(
+    "chrome://loop/content/modules/loop-content-process.js", true);
```

Comment:
Done.  It's not obvious to me whether the randomization hack is needed, but I put it in anyway since it can't really hurt us.